scenes/Locker.gd

```diff
@@ -22,7 +22,6 @@ var lock_enabled = true
 func _ready():
 #	for i in range(3):
 #		tumblers[i].rotation_degrees.x = current_combination[i] * 36
-	tween.start()
```

Comment:
After removing this line the `_ready` method is empty and the game crashes. Apart from this everything works as expected.